tests/pipeline-all-amps/repo/run-repo.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+PROJECT_NAME=repo
+[[ "$BUILD_ENABLED" == "true" ]] && mvn clean install
+env \
+	DOCKER_IMAGE_REPO=alfresco/alfresco-pipeline-all-amps-$PROJECT_NAME \
+	../run.sh "$@"
```

Comment:
It took me a while to work out what this script was doing, and I'm still not sure who we're expecting to use it.  Please could you add a comment (or `-h` option would be better, but it slightly interferes with the use of `$@`) to explain how to use it?

Reply:
it was helpful for me to test as it uses plain docker rather than magic from maven plugins